GameServer/Logic/Chatbox/ChatboxManager.cs

```diff
@@ -157,9 +157,43 @@ public ChatCommand GetCommand(string commandString)
             {
                 return _chatCommandsDictionary[commandString];
             }
-            else
+
+            return null;
+        }
+
+        public void RemoveAllCommandsButHelp()
```

Comment:
How about we don't use this function, but instead, if cheats are disabled, never add the commands there in the first place?

I'd suggest adding a function called `LoadChatCommands` in this class, and only run it if cheats are enabled.